.travis.yml

```diff
@@ -17,7 +20,8 @@ notifications:
   email: false
 
 install:
-  - pip install --quiet -r requirements.txt
+  - pip install --quiet -e ".[notebooks]"
```

Comment:
should we just put all the requirements in the setup.py with no extras ?

Reply:
and use here pip install --quiet .

Reply:
I like the idea of having a "simple" version of scVI, and another one which includes almost all the tools needed to run the notebooks.
It looks like installing the scVI package with the extras would be really easy (scanpy does something similar):

```
pip install scvi[notebooks]
```
That being said I can get rid of this distinction if you want (full installation is not that long and introducing extras may be confusing to the users?)

Reply:
I think the regular pip install should have the minimum number of requirements to make scVI work. We should have these extras for stuff required in the notebooks, testing, and building the docs.

In the readme we can add the different pip install commands. scVI technically does not require scanpy, louvian, etc. so it's a bit weird to force installation of that even if they are extremely useful downstream of scVI.